lib/api/stat.js

```diff
@@ -0,0 +1,14 @@
+var stats = require('../tools/stats.js');
+var request = require('../tools/smb2-forge').request;
+
+module.exports = function stat(path, cb) {
+  var connection = this;
+  request('open', { path: path }, connection, function(err, file) {
+    if (err !== null) {
```

Comment:
Please, follow existing style :slightly_smiling_face: 

```suggestion
    if (err != null) {
```

Reply:
done.